xclim/core/calendar.py

```diff
@@ -70,19 +70,23 @@ def get_calendar(obj: Any, dim: str = "time") -> str:
     -------
     str
       The cftime calendar name or "default" when the data is using numpy's or python's datetime types.
+      Will always return "standard" instead of "gregorian", following CF conventions 1.9.
     """
     if isinstance(obj, (xr.DataArray, xr.Dataset)):
         if obj[dim].dtype == "O":
             obj = obj[dim].where(obj[dim].notnull(), drop=True)[0].item()
         elif "datetime64" in obj[dim].dtype.name:
             return "default"
-
-    obj = np.take(
-        obj, 0
-    )  # Take zeroth element, overcome cases when arrays or lists are passed.
+    elif isinstance(obj, xr.CFTimeIndex):
+        obj = obj.values[0]
+    else:
+        obj = np.take(obj, 0)
+        # Take zeroth element, overcome cases when arrays or lists are passed.
     if isinstance(obj, pydt.datetime):  # Also covers pandas Timestamp
         return "default"
     if isinstance(obj, cftime.datetime):
+        if obj.calendar == "gregorian":
+            return "standard"
```

Comment:
Perhaps this comment is out of scope, but would we want to emit a warning here about this change of calendar? I don't think we specify any output `Conventions` to our datasets. Should we be appending this information to our output Datasets?

Reply:
I think we do not add this attribute, simply because we don't output Datasets. If we ever do, than I guess we could append the info. And if we do, we'll have to be even more careful about this kind of change.

It's a good question. But I believe that this change doesn't need a warning : 
  1. CF conventions have always accepted "standard" as equivalent to "gregorian", they just dropped the latter recently.
  2. Code might "break", but only with cftime <= 1.5.1. For people using the forthcoming cftime 1.5.2, this break will be generalized and we will be able to shift responsability to them ;).

Reply:
If that's the case, is this an event where we should bump the required version of CFtime within xclim?

Reply:
It's not released yet! That bug was found because of our smart tox builds using the master!